src/routes/blog/index.xml.ts

```diff
@@ -53,7 +53,8 @@ function makeRssFeed(posts: Newsletter[]): string {
 }
 
 export async function get(): Promise<RequestHandlerOutput> {
-  return new Response(JSON.stringify(makeRssFeed(await fetchNewsletters())), {
+  const posts: Newsletter[] = (await fetchNewsletters()).posts;
```

Comment:
Nit: Fancy [_deconstructor syntax_](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) simplification.
```suggestion
  const { posts } = await fetchNewsletters();
```